inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -104,10 +106,39 @@ def simulate_async(
         )  # convert to fs and divide by the time step of the simulation (2 fs)
         self.integrator = integrator
         self.nsteps_minim = nsteps_minim
+
+        self.command_file = "commands_charged.json" if self.charged\
+                                else "commands_neutral.json"
         commands = self.read_commands_from_file(
             os.path.join(self.template_dir, "commands.json"))
         return super().simulate_async(simulator, commands=commands)
 
+    def compute_charge(self, simulator: Simulator = GROMACS()):
+        """Check if the protein is charged."""
+
+        output_dir = os.path.dirname(self.protein_pdb)
+
+        simulator.run(output_dir,
+                      commands=[{
+                          "cmd":
+                              "gmx pdb2gmx -f \
+                         protein.pdb -o protein.gro \
+                         -water tip3p -ff amber99sb-ildn",
+                          "prompts": []
+                      }],
```

Comment:
Do you want to move this to a file, for consistency with the `simulate()` method?